webapp/packages/core-authentication/src/UserConfigurationParametersResource.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2021 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+import { computed, makeObservable, runInAction } from 'mobx';
+
+import { injectable } from '@cloudbeaver/core-di';
+import { CachedMapAllKey, CachedMapResource, GraphQLService, isResourceKeyList, ResourceKey } from '@cloudbeaver/core-sdk';
+
+import { UserInfoResource } from './UserInfoResource';
+
+@injectable()
+export class UserConfigurationParametersResource extends CachedMapResource<string, any> {
+  get parametersAvailable() {
+    return this.userInfoResource.data !== null;
+  }
+
+  constructor(
+    private readonly graphQLService: GraphQLService,
+    private readonly userInfoResource: UserInfoResource,
+  ) {
+    super();
+
+    this.userInfoResource.authenticationChange.addHandler(data => {
+      if (data === 'logout') {
+        this.data.clear();
+      }
+    });
+
+    this.userInfoResource.onDataUpdate.addHandler(this.loadAll.bind(this));
+    this.userInfoResource.onDataOutdated.addHandler(this.markOutdated.bind(this));
```

Comment:
lets remove this resource and move `delete` function to `userInfoResource` as `deleteConfigurationParameter` and `add` function as `setConfigurationParameter`. Add `includeConfigurationParameters` to default includes. Also create function `getConfigurationParameter`